internal/hud/hud.go

```diff
@@ -137,35 +132,39 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 				} else {
 					logger.Get(ctx).Infof("no urls for resource '%s' ¯\\_(ツ)_/¯", selected.Name)
 				}
-			case r == 'q': // [Q]uit
-				h.Close()
-				dispatch(ExitAction{})
-				return true
-			case r == 'l': // [L]og
+			case r == 'p': // [P]od log
```

Comment:
the current implementation is this shows the build log if the last build failed, or the pod log if it didn't

Reply:
Good point. I'm not entirely happy with the key legend design. But I settled on this for now:

L key — Tilt logs
V key — view log (build or pod)

I don't *love* it but it's no longer actively misleading?